docs/attributes-registry/messaging.md

```diff
@@ -66,9 +68,9 @@ size should be used.
 |---|---|
 | `activemq` | Apache ActiveMQ |
 | `aws_sqs` | Amazon Simple Queue Service (SQS) |
-| `azure_eventgrid` | Azure Event Grid |
-| `azure_eventhubs` | Azure Event Hubs |
-| `azure_servicebus` | Azure Service Bus |
+| `eventgrid` | Azure Event Grid |
+| `eventhubs` | Azure Event Hubs |
+| `servicebus` | Azure Service Bus |
 | `gcp_pubsub` | Google Cloud Pub/Sub |
```

Comment:
just thinking ahead, for consistency would we rename `aws_sqs` and `gcp_pubsub` too?

Reply:
I don't think so, but I create an overarching issue to discuss https://github.com/open-telemetry/semantic-conventions/issues/608 - we have `db.dynamodb` or `db.spanner` and it's not clear if there is any reasonable pattern.

The reason to rename these is because they are de-facto called `servicebus` and `eventhubs` and never had `azure_` prefix in the instrumentation.

We would prefer to not have `azure_` (and if we have to, we'd rather do `azure.servicebus`), if otel would create some guidance requiring it.

Reply:
> I create an overarching issue to discuss https://github.com/open-telemetry/semantic-conventions/issues/608

👍